pkg/console/subresource/configmap/configmap.go

```diff
@@ -3,6 +3,10 @@ package configmap
 import (
 	"fmt"
 
+	yaml2 "github.com/ghodss/yaml"
```

Comment:
This is a wrapper around the `gopkg.in/yaml.v2` we are using below. Had checked the [docs](https://github.com/ghodss/yaml#compatibility) and should be compatible with it, so there shouldn't be need for keeping both **yaml** libs

Reply:
Ha, my bad.  Didn't look at my imports to realize I imported a second yaml package.  Thanks!

Reply:
Actually, compatible but not the same interface available.

`yaml2.JSONToYAML()` is something we gain.  I could potentially look for a way to do this manually, but I'm not sure I want to fiddle with that.

Reply:
Yeah If the interface is different we can maybe just add a "techdebt" comment, that in case we got time we can update, but would really pay attention if its time-consuming.

Reply:
`yaml` lib has funcs for creating `MapSlice`, `MapItem`, and a bunch of others.  Its a good core YAML lib with a lot of features.  `yaml2` is pretty small (300 lines), and uses `yaml` itself under the hood.  It might just be poorly named for a set of additional helper funcs to the main `yaml` lib.

I don't think we can drop `yaml`, we can't swap `yaml2` for `yaml`, we'd end up just dropping `yaml2` and writing the funcs ourselves.